README.md

```diff
@@ -1,6 +1,6 @@
 # icub-models
 
-Repository containing models automatically generated from the CAD file by [icub-model-generator](https://github.com/robotology-playground/icub-model-generator).
+Repository containing models [automatically](https://github.com/robotology-playground/icub-model-generator/blob/master/.travis.yml#L76) generated from the CAD file by [icub-model-generator](https://github.com/robotology-playground/icub-model-generator).
```

Comment:
The problem in linking in a specific line of a file committed to a repo is that when the travis file is updated (and the line 76 changes!) the link will not be updated. Not a big problem, but it is worth to consider it.

Reply:
I always wondered whether there was a way to get a permanent link to the line :grin:. Maybe linking to a specific commit version?

Reply:
Yes, that works but you still have a similar problem if the travis script changes completely: you are not linking the current version anymore. Anyway, I think that line does its job for now.